src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
-export interface Figure {
 
+export interface Figure {
+  shape: string,
+  color: string,
+  area: number,
+  getArea(a: number, b: number, c: number): void,
 }
 
 export class Triangle {
+  constructor(
+    public shape: string,
+    public color: string,
+    public area: number,
+  ) {
+    this.shape = shape;
+    this.color = color;
+  }
+
+  getArea(a: number, b: number, c: number): number {
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/223662879-d5f55ba7-1fac-480a-a3fd-1acbf6647091.png)

Reply:
Not fixed. getArea functions shouldn't accept any parameters